main.tf

```
@@ -44,7 +44,14 @@ resource "aws_security_group_rule" "egress" {
   security_group_id = join("", aws_security_group.default.*.id)
 }
 
-resource "aws_rds_cluster" "primary" {
+resource "aws_cloudwatch_log_group" "default" {
+  count             = var.enabled && length(var.enabled_cloudwatch_logs_exports)>0 ? 1 : 0
+  name              = "/aws/rds/cluster/${coalesce(join("", aws_rds_cluster.default.*.id), join("", aws_rds_cluster.secondary.*.id))}/postgresql"
```

I am not completely sure how log groups work with primary and secondary clusters. According to [this documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.CloudWatch.html#AuroraPostgreSQL.CloudWatch.Monitor) you need a log group per log type anyway.

```suggestion
  for_each          = module.this.enabled ? toset(var.enabled_cloudwatch_logs_exports) : []
  name              = "/aws/rds/cluster/${coalesce(join("", aws_rds_cluster.primary.*.id), join("", aws_rds_cluster.secondary.*.id))}/${each.value}"
```

Also, update description of `var.enabled_cloudwatch_logs_exports` to include `postgresql` in the list of valid log types.